README.md

```diff
@@ -1860,13 +1860,16 @@ The Answers SDK exposes a `formatRichText` function which translates CommonMark
 ensure that a Rich Text Formatted value is shown properly on the page. To use this function, call it like so:
 
 ```js
-ANSWERS.formatRichText(rtfFieldValue, eventOptionsFieldName)
+ANSWERS.formatRichText(rtfFieldValue, target, eventOptionsFieldName)
```

Comment:
would we want to generalize this to an attributes param, that lets you stick arbitrary attributes onto the html, or is that overengineered?

Reply:
We currently don't have requirements for that. The HHs are really only interested in changing the 'target' attribute of a link.